### PR TITLE
Update training-external-resources.md

### PR DESCRIPTION
**Pull request by suzannewoodward**
Added row for microsoft enterprise skills initiative. Please review.

### CODE REVIEW

**Comment by Pilargit12** [comment]
Hi @mtspn ! I was wondering if you could help vet this external resource training suggestion to add.  If allowed, I can review the text and then merge to main.  Thank you for your help!  

Also @suzannewoodward  thank you for your suggestion!

**Comment by mtspn** [comment]
Thanks again for the suggestion @suzannewoodward 

Typically this page hosts training links relating to the OpenShift private cloud. Have you taken courses through the Microsoft ESI? Would you say that they're general enough about cloud development that they'd also be relevant for devs working on OpenShift? Are there any particular courses on there you would recommend? 

I'd definitely like to share this link, but just trying to make sure this is the best place! Maybe your or I can even promote this at a community meetup.

**Comment by suzannewoodward** [comment]
Hi Matt,

Here's the screenshot from the platform community meetup where folks were saying that this was a helpful resource for training. If you'd like me to reach out to these folks to get feedback about the training quality and any recommended courses please let me know.

Thanks
Suzanne

***@***.***

Suzanne Woodward
DevOps Chapter Lead, Development & Digital Services
Water, Land & Resource Stewardship | NRIDS
***@***.******@***.***>
Work: 236-478-2843

File: src/docs/training-and-learning/training-external-resources.md [hunk 1]
```diff
@@ -41,6 +41,8 @@ The resources below are from sources outside of the B.C. government. They provid
 
 [The Twelve-Factor App](https://12factor.net/) - by Heroku (Adam Wiggins)
 
+[Microsoft Enterprise Skills Initiative](https://esi.microsoft.com/) by Microsoft. Free for BC Government employees through the contract between The Province and Microsoft. Useful for training and then proceeding to write certification exams to get Azure certified, 50% off exam costs as well.
```
Comment (mtspn) [inline src/docs/training-and-learning/training-external-resources.md:44]:
```suggestion
[Microsoft Enterprise Skills Initiative](https://esi.microsoft.com/) - by Microsoft. Free for BC Government employees through the contract between The Province and Microsoft. Useful for training and then proceeding to write certification exams to get Azure certified, 50% off exam costs as well. The Microsoft Azure Fundamentals course contains some broad cloud fundamentals that apply to OpenShift and other cloud technologies. 
```

**Review by mtspn — APPROVED** [review]
Looks good to me @Pilargit12

File: src/docs/training-and-learning/training-external-resources.md [hunk 2]
```diff
@@ -41,6 +41,8 @@ The resources below are from sources outside of the B.C. government. They provid
 
 [The Twelve-Factor App](https://12factor.net/) - by Heroku (Adam Wiggins)
 
+[Microsoft Enterprise Skills Initiative](https://esi.microsoft.com/) - by Microsoft. Free for BC Government employees through the contract between The Province and Microsoft. Useful for training and then proceeding to write certification exams to get Azure certified, 50% off exam costs as well. The Microsoft Azure Fundamentals course contains some broad cloud fundamentals that apply to OpenShift and other cloud technologies.
```
Comment (Pilargit12) [inline src/docs/training-and-learning/training-external-resources.md:44]:
```suggestion
[Microsoft Enterprise Skills Initiative](https://esi.microsoft.com/) - by Microsoft. Free for B.C. government employees. Useful for training and take certification exams to get Azure certification. The exam has a 50% off discount. The Microsoft Azure Fundamentals course contains broad cloud fundamentals that apply to OpenShift and other cloud technologies. 
```

**Review by Pilargit12 — DISMISSED** [review]

File: src/docs/training-and-learning/training-external-resources.md (hunk 2 above)
Comment (mtspn) [reply src/docs/training-and-learning/training-external-resources.md:44]:
```suggestion
[Microsoft Enterprise Skills Initiative](https://esi.microsoft.com/) - by Microsoft. Free for B.C. government employees through the contract between The Province and Microsoft. Useful for training and certification exams. The exam has a 50% off discount. The Microsoft Azure Fundamentals course contains some broad cloud fundamentals that apply to OpenShift and other cloud technologies. 
```

**Pull request merged**